modules/platform/nuxeo-platform-query-api/src/main/java/org/nuxeo/ecm/platform/query/api/AbstractPageProvider.java

```diff
@@ -355,7 +355,6 @@ public void addQuickFilter(QuickFilter quickFilter) {
     public void setSortInfos(List<SortInfo> sortInfo) {
         if (sortInfoChanged(this.sortInfos, sortInfo)) {
             this.sortInfos = sortInfo;
-            refresh();
```

Comment:
Are we sure of the impacts other than related to the PaginableAdapter?

Reply:
I asked my self the same question, 

At the beginning To minimise this impact, I created a new method that use a boolean param, to refresh at demande. But after that some tests fails on my local machine (test where  index = `1`). Because the tests was a false positive, as each set of sort reset the index and I had some Page Provider Cache issue...

To be honest, the TP passe on default and mongoDB. But perhpas it will impact others things. But I don't know why we call refresh when set the sorter because this method will reset index and do other things.

To avoid any breaking changes if any. We can override the setSorter for the `AuditPageProvider` to avoid refresh only for this provider.